### ISSUE TITLE
Fix external anchor links

### ISSUE THREAD

**Issue by Mimaaa** [part-up/part-up#997]
In general using `target="blank"` is not the best idea because of smartphone native controls (swiping back to the last page), but it's still often used to retain users instead of closing your own website and providing them with a possibly more interesting one.

I'm not going to remove the tag because it's there for a purpose, but the tag is not safe and reliable by its own. See the paragraph below.

> When your page links to another page using target="_blank", the new page runs on the same process as your page. If the new page is executing expensive JavaScript, your page's performance may also suffer.
>On top of this, target="_blank" is also a security vulnerability. The new page has access to your window object via window.opener, and it can navigate your page to a different URL using window.opener.location = newURL.

See: [this article](https://developers.google.com/web/tools/lighthouse/audits/noopener)

What I'm going to do is add the `rel="noopener"` to the outgoing links.

**Comment by ralphboeije** [comment]
Hi @Mimaaa thanks for pointing this out. Wasn't aware of the security issue. Choice for target blank is deliberate, even knowing the experience limitations.

**Comment by Mimaaa** [comment]
First and foremost: Part-Up is using the [Autolinker.js package](https://github.com/gregjacobs/Autolinker.js) with version 0.18.1. They are currently at version 1.4.2. The latest version also includes the [fix above](https://github.com/gregjacobs/Autolinker.js/issues/149), which is important and also will automatically fix most outgoing links. 

I've just created a pull request with all the links that I've been able to edit manually.
